DataFormats/Provenance/interface/BranchChildren.h

```diff
@@ -6,7 +6,8 @@
 BranchChildren: Dependency information between branches.
 
 ----------------------------------------------------------------------*/
-#if !defined(DataFormats_Provenance_ProductDependencies_h)
+#if (not defined __INCLUDE_LEVEL__ or __INCLUDE_LEVEL__ > 0) and \
```

Comment:
Both clang and gcc define `__INCLUDE_LEVEL__`, but MSVC doesn't.
The extra check is just to be safe in case other compilers have issues with it.